<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>info</class>
 <widget class="QDialog" name="info">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>-20</y>
     <width>1280</width>
     <height>720</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/Ss/info.png);
background-color: rgba(255, 255, 255, 0);
color: rgba(255, 255, 255, 0);</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>720</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255, 255, 255, 0);
color: rgba(255, 255, 255, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>640</y>
     <width>201</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	
	color: rgb(255, 255, 255);
	
	font: 16pt bold &quot;Candara&quot;;
	background-color: rgb(37, 30, 59);
  border: 3px rgb(31, 31, 31);
 border-radius:25px;
}
QPushButton:pressed {
    
	
	
	background-color: rgb(126, 36, 102);
}
</string>
   </property>
   <property name="text">
    <string>Понятно</string>
   </property>
  </widget>
  <widget class="QPushButton" name="dark">
   <property name="geometry">
    <rect>
     <x>1120</x>
     <y>570</y>
     <width>111</width>
     <height>111</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{background-color: rgba(255, 255, 255, 111);
background-image: url(:/Ss/switchnight.png);
border-radius: 55px;
}
QPushButton:hover
{
	background-color: rgba(128, 214, 250,22);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>720</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(74, 12, 13, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>label_2</zorder>
  <zorder>label_3</zorder>
  <zorder>label</zorder>
  <zorder>dark</zorder>
  <zorder>pushButton</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
